cmake/CMakejucer.cmake

```diff
@@ -121,6 +121,10 @@ function(jucer_project_end)
 
   if(APPLE)
     target_compile_options(${JUCER_PROJECT_NAME} PRIVATE -std=c++11)
+
+    if(NOT DEFINED CMAKE_BUILD_TYPE)
+      set(CMAKE_BUILD_TYPE "Debug")
+    endif()
```

Comment:
@MartyLake please check that this change doesn't break anything when configuring for Xcode.